src/App.css

```diff
@@ -673,8 +673,8 @@ h1 .num-contexts {
 }
 
 .mobile .child.leaf > .bullet::before {
-  left: -0.05em;
-  top: 0.03em;
+  left: -2px;
+  top: 3.1;
```

Comment:
Your changes introduce a regression on desktop and iPhone, where the current styles work as desired. Thus you will need to add your styles to an Android-specific selector.

e.g. instead of changing the styleset for `.mobile .child.leaf > .bullet::before`, you would add new styles to `.android .child.leaf > .bullet::before`. `.android` can be added in [AppComponent](https://github.com/cybersemics/em/blob/dev/src/components/AppComponent.js#L83-L90).

Unfortunately the spacing issues are browser-specific, so we have to have different selectors for each user agent, unless there is a universal solution I am not aware of.

Reply:
Hi @raineorshine the new committing. Should not cause a regression. Since the app is primarily been deployment and testing in iOS environment I have decide to revert the first properties changes and instead make a section fully on android so we handle the bullets and test in different properties a long the way for Android only.